packages/manager/cypress/e2e/core/helpAndSupport/close-support-ticket.spec.ts

```diff
@@ -0,0 +1,164 @@
+import 'cypress-file-upload';
+import {
+  mockAppendFeatureFlags,
+  mockGetFeatureFlagClientstream,
+} from 'support/intercepts/feature-flags';
+import { makeFeatureFlagData } from 'support/util/feature-flags';
+import { ui } from 'support/ui';
+import {
+  randomItem,
+  randomLabel,
+  randomNumber,
+  randomPhrase,
+} from 'support/util/random';
+import { supportTicketFactory } from 'src/factories';
+import {
+  mockGetSupportTicket,
+  mockGetSupportTickets,
+  mockGetSupportTicketReplies,
+  mockCloseSupportTicket,
+} from 'support/intercepts/support';
+import { SEVERITY_LABEL_MAP } from 'src/features/Support/SupportTickets/constants';
+import { cleanUp } from 'support/util/cleanup';
+import { authenticate } from 'support/api/authentication';
+import {
+  closableMessage,
+  closeButtonText,
+} from 'support/constants/help-and-support';
+
+describe('close support tickets', () => {
+  after(() => {
+    cleanUp(['linodes']);
+  });
+
+  authenticate();
+
+  /*
+   * - Opens a Help & Support ticket with mocked ticket data.
+   * - Confirms that there is no "close ticket" button showing up for the default support ticket.
+   */
+  it('cannot close a default support ticket by customers', () => {
+    const mockTicket = supportTicketFactory.build({
+      id: randomNumber(),
+      summary: randomLabel(),
+      description: randomPhrase(),
+      severity: randomItem([1, 2, 3]),
+      status: 'new',
+    });
+
+    // Get severity label for numeric severity level.
+    // Bail out if we're unable to get a valid label -- this indicates a mismatch between the test and source.
+    const severityLabel = SEVERITY_LABEL_MAP.get(mockTicket.severity!);
+    if (!severityLabel) {
+      throw new Error(
+        `Unable to retrieve label for severity level '${mockTicket.severity}'. Is this a valid support severity level?`
+      );
+    }
+
+    mockAppendFeatureFlags({
+      supportTicketSeverity: makeFeatureFlagData(true),
+    });
+    mockGetFeatureFlagClientstream();
+    mockGetSupportTickets([mockTicket]);
+    mockGetSupportTicket(mockTicket).as('getSupportTicket');
+    mockGetSupportTicketReplies(mockTicket.id, []).as('getReplies');
+
+    cy.visitWithLogin('/support/tickets');
+
+    // Confirm that tickets are listed as expected.
+    cy.findByText(mockTicket.summary).should('be.visible').click();
+
+    cy.wait(['@getSupportTicket', '@getReplies']);
+
+    cy.url().should('endWith', `/tickets/${mockTicket.id}`);
+    cy.findByText(
+      mockTicket.status.substring(0, 1).toUpperCase() +
+        mockTicket.status.substring(1)
+    ).should('be.visible');
+    cy.findByText(`#${mockTicket.id}: ${mockTicket.summary}`).should(
+      'be.visible'
+    );
+    cy.findByText(mockTicket.description).should('be.visible');
+    cy.findByText(severityLabel).should('be.visible');
+
+    // Confirm that the support ticket is not closable by default.
+    cy.findByText(closableMessage).should('not.exist');
```

Comment:
```suggestion
    cy.findByText(closableMessage, { exact: false }).should('not.exist');
```
We need `{exact: false}` here otherwise this assertion would pass even if the closable message appears!